cmake/external/boringssl.cmake

```diff
@@ -21,14 +21,14 @@ endif()
 set(patch_file 
   ${CMAKE_CURRENT_LIST_DIR}/../../scripts/git/patches/boringssl/0001-disable-C4255-converting-empty-params-to-void.patch)
 
-set(commit_tag 83da28a68f32023fd3b95a8ae94991a07b1f6c62)
+set(boringssl_commit_tag 83da28a68f32023fd3b95a8ae94991a07b1f6c62)
 
 ExternalProject_Add(
   boringssl
   
   DOWNLOAD_COMMAND 
     COMMAND git init boringssl
-    COMMAND cd boringssl && git fetch --depth=1 https://github.com/google/boringssl ${commit_tag} && git reset --hard FETCH_HEAD
+    COMMAND cd boringssl && git fetch --depth=1 https://github.com/google/boringssl ${boringssl_commit_tag} && git reset --hard FETCH_HEAD
```

Comment:
Should this be https//github.com/google/boringssl.git?

Reply:
Seems like it works, so no?